stats.jenkins.io.tf

```diff
@@ -0,0 +1,37 @@
+resource "azurerm_resource_group" "stats_jenkins_io" {
+  name     = "stats-jenkins-io"
+  location = var.location
+  tags     = local.default_tags
+}
+
+resource "azurerm_storage_account" "stats_jenkins_io" {
+  name                      = "statsjenkinsio"
+  resource_group_name       = azurerm_resource_group.stats_jenkins_io.name
+  location                  = azurerm_resource_group.stats_jenkins_io.location
+  account_tier              = "Standard"
```

Comment:
Note: if you use a Standard storage account, you'll have to check the cost due to amount of transactions. It usually are the write which are costly: what would be the statistic's update frequency?

I believe we can for it but you have to watch the costs of this resource group in the upcoming (4-6) weeks